README.md

```diff
@@ -102,7 +124,7 @@ conan install timsort/2.0.2
 
 ## DIAGNOSTICS & INFORMATION
 
-A few configuration macros allow gfx::timsort to emit diagnostic, which might be helpful to diagnose issues:
+A few configuration macros allow gfx::timsort and gfx::timmerge to emit diagnostic, which might be helpful to diagnose issues:
```

Comment:
Looks like I forgot to use code font for `gfx::timsort` there, but if possible I would rather use code font.

Reply:
Done: "`gfx::timsort` and `gfx::timmerge`".